### DISCUSSION TITLE
Removal of comments in version 1.43.3

### DISCUSSION THREAD

**Discussion by AnnaMatlapudi** [denoland/deno#23814] [Q&A]
Does anyone know what is happening here with new version (1.43.3) when a function code is displayed with `toString()`, comments are removed. Whereas until previous version (1.42.2) comments are retained (which is the case with node(js) runtime also).

Is there a change in default configuration causing this? if so, what are my options to switch back to behavior of keeping comments? Please help

**Test Code:**
```typescript
const utils = {
  fx1: () =>
    // This is addition
    123 + 567,
  fx2: () =>
    // This is subtraction
    /* one more line of comment */
    123 + 567,
};

const fn = utils["fx2"];
console.log(fn.toString());
```
**Output from `deno 1.43.3`** (release, x86_64-unknown-linux-gnu) , `v8 12.4.254.13` and `typescript 5.4.5`
```typescript
()=>123 + 56
```
**Output from `deno  1.42.2`** (release, x86_64-unknown-linux-gnu) , `v8 12.3.219.9` and `typescript 5.4.3`
```typescript
()=>// This is subtraction
    /* one more line of comment */ 123 + 567
```

**Output from `node v21.5.0`**
```JavaScript
() =>
    // This is subtraction
    /* one more line of comment */
    123 + 567
```

**Comment by bartlomieju** [comment]
It appears this is only happening in the REPL or if you use TypeScript. IIRC @dsherret did a change that removes all comments during transpilation some time ago, but I'm not 100% sure.

**Reply by dsherret** [reply]
Yeah, I think we need to revert that.

**Reply by dsherret** [reply]
I opened https://github.com/denoland/deno/pull/23815
